learning/step_by_step/your_first_game.rst

```diff
@@ -324,6 +324,7 @@ the player when starting a new game.
         position = pos
         show()
         monitoring = true
```

Comment:
You can delete this line as well. The original code using `monitoring` was written before the `disabled` property was working in an earlier alpha.